README.md

```diff
@@ -31,30 +32,40 @@
 
-## Creating a KV Database Instance
+## Store your Chat History in Supabase
 
-Follow the steps outlined in the [quick start guide](https://vercel.com/docs/storage/vercel-kv/quickstart#create-a-kv-database) provided by Vercel. This guide will assist you in creating and configuring your KV database instance on Vercel, enabling your application to interact with it.
+Go to [supabase.com](https://supabase.com/) and click "Start your project" in the top right to create a new project. When your project is created, grab your api keys. Click the Connect button in the top right of the project homepage -> App Frameworks -> Next.js. Copy the contents of the `.env.local` file with the api keys and add this to your local `.env.local` file.
 
-Remember to update your environment variables (`KV_URL`, `KV_REST_API_URL`, `KV_REST_API_TOKEN`, `KV_REST_API_READ_ONLY_TOKEN`) in the `.env` file with the appropriate credentials provided during the KV database setup.
+## Get an OpenAI API key
+
+Go to the [OpenAI Platform](https://platform.openai.com/api-keys) and login or signup, and create a new secret key. Add this key to your `.env.local` file as `OPENAI_API_KEY`.
 
 ## Running locally
 
-You will need to use the environment variables [defined in `.env.example`](.env.example) to run Next.js AI Chatbot. It's recommended you use [Vercel Environment Variables](https://vercel.com/docs/projects/environment-variables) for this, but a `.env` file is all that is necessary.
+You will need to use the environment variables [defined in `.env.example`](.env.example) to run Next.js AI Chatbot. It's recommended you use [Vercel Environment Variables](https://vercel.com/docs/projects/environment-variables) for this, but a `.env.local` file is all that is necessary.
 
-> Note: You should not commit your `.env` file or it will expose secrets that will allow others to control access to your various OpenAI and authentication provider accounts.
+> Note: You should not commit your `.env.local` file or it will expose secrets that will allow others to control access to your various OpenAI and authentication provider accounts.
 
 1. Install Vercel CLI: `npm i -g vercel`
 2. Link local instance with Vercel and GitHub accounts (creates `.vercel` directory): `vercel link`
 3. Download your environment variables: `vercel env pull`
 
+Next, setup your Supabase project to store your Chat History.
+
+1. In `/supabase/migrations/20240916140519_init.sql`, you'll find a SQL script to setup your database tables in Supabase
+2. Copy this script, and visit the [SQL Editor](https://supabase.com/dashboard/project/zlarqpgkbjqgrvuquugu/sql/new) of the project you just created.
```

Comment:
Needs to be ..

Go to Vercel Storage, click on "open in supabase", go to SQL editor ...